src/vue/forms/InvestForm.vue

```diff
@@ -0,0 +1,564 @@
+<template>
```

Comment:
if your component takes 564 rows, then you should think about splitting it on smaller ones. Now it tries to handle too much logic inside it

Reply:
@quinn71 You right but right now we have not so much time for it. I created a task for it: